### ISSUE TITLE
findobjs.find_objects() returning more than expected

### ISSUE THREAD

**Issue by walker-tx** [pmaupin/pdfrw#156]
Howdy - want to start off saying that I've really enjoyed using this library, its much simpler than some of the others that I've tried!

**TLDR:**
I have a source PDF with two markups ('/Type/Annot') that I would like to find and isolate from the rest of the objects in the document. Though the markups are included in the result, the printed list totals about 150 items. _This is likely due to a misunderstanding on my behalf._

**My code**
```Python

#This is just what I've thrown together to test out and learn about findobjs.find_objects().

import os

from pdfrw import PdfName, PdfReader, findobjs


def main():
    source = 'C:\\Users\\wlockiv\\Desktop\\1128686_B4_P101A-P101B_WATR_1_IFC_2018.08.21.pdf'
    source = PdfReader(source).pages[0]
    foundobjects = list(findobjs.find_objects(source, valid_types=(
        PdfName.Annot, None), valid_subtypes=('/FreeText', None)))
    print(foundobjects)
    print(len(foundobjects))


if __name__ == '__main__':
    main()

````

**The objects I would like to see in the list:**
```
First markup, a text box containing the text "AMARKUP"

obj<</Subj(Text Box)/Type/Annot/P 160 0 R/F 4/C[]/CreationDate(D:20181027073019-05'00')/T(wlockiv)/Subtype/FreeText/BS<</W 0/S/S/Type/Border>>/DA(1 0 0 rg /Helv 12 Tf)/DS(font: Helvetica 12pt; text-align:left; margin:3pt; line-height:13.8pt; color:#FF0000)/M(D:20181027074432-05'00')/Rotate 270/AP<</N 174 0 R>>/Rect[2039.035 2657.21 2059.035 2724.55]/NM(TGNSAFEHIPUOWNHO)/RC(<?xml version="1.0"?><body xmlns:xfa="http://www.xfa.org/schema/xfa-data/1.0/" xfa:contentType="text/html" xfa:APIVersion="BluebeamPDFRevu:2018" xfa:spec="2.2.0" style="font:Helvetica 12pt; text-align:left; margin:3pt; line-height:13.8pt; color:#FF0000" xmlns="http://www.w3.org/1999/xhtml"><p>AMARKUP</p></body>)/Contents(AMARKUP)>>
```
```
Second markup, cloud shape

obj<</Subj(Cloud)/Type/Annot/P 160 0 R/F 4/C[1 0 0]/CreationDate(D:20181027074408-05'00')/T(wlockiv)/Subtype/Polygon/M(D:20181027074408-05'00')/BE<</S/C/I 2>>/IT/PolygonCloud/AP<</N 422 0 R>>/Vertices[2025.342 2314.65 1992.06 2314.65 1992.06 2203.71 2025.342 2203.71]/Rect[1976.707 2188.357 2040.695 2330.003]/NM(GXCSTGKMCIWITSMU)>>
```

**Description of Overall Goal:**

I'm working on a script that will copy construction documents (ie - drawings/blueprints) from a source location to a destination location. The source documents are private, and the resulting ones are public and meant for collaboration and markup. If a revision has been made to the source document, the script will recognize its obsolete clone in the public location and overwrite it with the latest revision of the source. I would like to make sure any markups on the public document don't get lost, bust instead get saved and applied to their same locations in the new public revision.

If findobjs isn't the right choice for going about preserving those markups, I'm open to suggestions!

My repo is [here](https://gitlab.com/wlockiv/dashsync), I'm a neophite at this kind of thing so please forgive my disorganization.
